app/services/gias/csv_transformer.rb

```diff
@@ -1,5 +1,5 @@
 module Gias
-  class CsvTransformer
+  class CSVTransformer
     include ServicePattern
```

Comment:
This service should now inherit from ApplicationService, and include ServicePattern removed

Reply:
I've moved the `ApplicationService` introduction into https://github.com/DFE-Digital/itt-mentor-services/pull/984.